app/controllers/hiring_staff/vacancies/job_location_controller.rb

```diff
@@ -13,7 +13,7 @@ def show
   def create
     @form.vacancy.readable_job_location = readable_job_location(@form.vacancy.job_location)
     store_vacancy_attributes(@form.vacancy.attributes)
-    session[:organisation_id] = current_organisation.id
+    session[:vacancy_attributes]['organisation_id'] = current_organisation.id
```

Comment:
I believe Cesi fixed this in the opposite direction in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1955  : he changed the code that reads this session value, but you changed the code that stores it.
```
-       @form.vacancy.organisation_vacancies.build(organisation_id: session[:vacancy_attributes]['organisation_id'])
+.      @form.vacancy.organisation_vacancies.build(organisation_id: session[:organisation_id])
```

Reply:
Either way seems fine

Reply:
Yours slightly neater

Reply:
Also, if it's not stored in vacancy_attributes it leaks when creating new job listings

Reply:
Actually, since you have written `.except('organisation_id', 'organisation_ids'` twice (in the code I've so far seen), I think Cesi's way is neater.

Reply:
See above

Reply:
But as you prefer

Reply:
I mean that issue could be fixed a different way, but I think I prefer it there